testcafe/service.js

```diff
@@ -4,20 +4,30 @@ import EditResourcePage from './pages/EditResourcePage';
 const servicePage = new ServicePage();
 const editResourcePage = new EditResourcePage();
 
-// Define service id to navigate to
-// Tests depend on service id
-const serviceId = 5;
+// Define service information to test
+// Tests depend on service information
+// TODO: Create a new service and add all this information in the future
+//  to not rely on hardcoded data
```

Comment:
Looks like most of the tests are failing because this data doesn't match. You probably want to update this part of the DB fixture populator to populate the same values: https://github.com/ShelterTechSF/askdarcel-api/blob/518efa307db3c7d4cc1ef00831358b0640dabfdd/lib/sheltertech/db/fixture_populator.rb#L148

Reply:
Ah, I was going off of the data we populate in the our local DBs. I will put up an MR for the testcafe data

Reply:
Change of plans; since we already had a test as to whether adding a service to a resource worked, I just extended that test to also verify the information on the service page.